src/Features/Traits/HttpResponse.php

```diff
@@ -0,0 +1,252 @@
+     */
+    public function aResponseHTMLShouldContainCSRFToken(): void
+    {
+        $converter = new CssSelectorConverter();
+        $xPath = $converter->toXPath("form input[name=\"_token\"]");
+
+        $crawler = new Crawler($this->response->getContent());
+        $node = $crawler->filterXPath($xPath)->first();
+
+        Assert::assertNotNull($node?->attr("value"));
+    }
+
+    /**
+     * @Then the response should be JSON
+     */
+    public function aResponseShouldBeJson(): void
+    {
+        Assert::assertJson($this->response->getContent());
+    }
+
+    /**
+     * @Then the response should contain JSON:
+     */
+    public function aResponseShouldContainJson(TableNode $table): void
+    {
+        $json = json_decode($this->response->getContent(), true);
+
+        foreach ($table->getRowsHash() as $key => $value) {
+            Assert::assertArrayHasKey($key, $json);
+            Assert::assertEquals($value, $json[$key]);
+        }
+    }
+
+    /**
+     * @Then the response should contain a JSON fragment:
+     */
+    public function aResponseShouldContainJsonFragment(TableNode $table): void
+    {
+        $json = json_decode($this->response->getContent(), true);
+
+        foreach ($table->getRowsHash() as $key => $value) {
+            Assert::assertArrayHasKey($key, $json);
+            Assert::assertStringContainsString($value, $json[$key]);
+        }
+    }
+
+    /**
+     * @Then the response should have cookie :name
+     */
+    public function aResponseShouldHaveCookie(string $name): void
+    {
+        Assert::assertTrue($this->response->headers->has("Set-Cookie"));
+        $cookies = $this->response->headers->getCookies();
+        $cookieNames = array_map(fn($cookie) => $cookie->getName(), $cookies);
+        Assert::assertContains($name, $cookieNames);
+    }
+
+    /**
+     * @Then the response should have header :header
+     */
+    public function aResponseShouldHaveHeader(string $header): void
+    {
+        Assert::assertTrue($this->response->headers->has($header));
+    }
+
+    /**
+     * @Then the response should have header :header equal to :value
+     */
+    public function aResponseShouldHaveHeaderEqualTo(string $header, string $value): void
+    {
+        Assert::assertEquals($value, $this->response->headers->get($header));
+    }
+
+    /**
+     * @Then the response should have status :status
+     */
+    public function aResponseShouldHaveStatus(int $status): void
+    {
+        Assert::assertEquals($status, $this->response->getStatusCode());
+    }
+
+    /**
+     * @Then the response should be OK
+     */
+    public function aResponseShouldBeOk(): void
+    {
+        Assert::assertEquals(200, $this->response->getStatusCode());
+    }
+
+    /**
+     * @Then the response should be created
+     */
+    public function aResponseShouldBeCreated(): void
+    {
+        Assert::assertEquals(201, $this->response->getStatusCode());
```

Comment:
```suggestion
        Assert::assertEquals(Response::HTTP_CREATED, $this->response->getStatusCode());
```
Maybe use constants instead